pkg/user/configure_user_command_test.go

```diff
@@ -0,0 +1,57 @@
+package user
+
+import (
+	"testing"
+
+	"github.com/landoop/lenses-go/pkg/api"
+	config "github.com/landoop/lenses-go/pkg/configs"
+	test "github.com/landoop/lenses-go/test"
+	"github.com/stretchr/testify/assert"
+)
+
+const contextOutput = "[master] [valid, current]\n{\n  \"host\": \"http://domain.com:80\",\n  \"token\": \"****\",\n  \"timeout\": \"15s\",\n  \"debug\": true,\n  \"basic\": {\n    \"username\": \"user\",\n    \"password\": \"****\"\n  }\n}\n"
+
+func TestContextCommandSuccess(t *testing.T) {
```

Comment:
we cover only the happy cases. It will be nice to test also bad cases. Also check for table driven testing in go where you can test same method for different test cases. an example here https://github.com/lensesio-dev/cloud-platform/blob/master/server/deployer/test/rest_test.go#L47